### ISSUE TITLE
No exceptions thrown when executing group by query without given an aggregation function in select clause

### ISSUE THREAD

**Issue by SteveYurongSu** [apache/iotdb#2564]
**Describe the bug**
No exceptions thrown when executing group by query without given an aggregation function in select clause.

**To Reproduce**
```sql
select ZT11709 from root.group_69.1701 GROUP BY([2020-06-09T23:51:35.267+08:00, 2020-06-17T23:17:14.887+08:00), 5m) 
```

**Expected behavior**
Should throw an exception and give a hint.

**Comment by Alima777** [comment]
Fixed.

**Comment by SteveYurongSu** [comment]
Fixed in [IOTDB-1396] #3267 .
